treeherder/model/management/commands/cycle_data.py

```diff
@@ -442,6 +443,89 @@ def _find_ideal_chunk_size(self) -> int:
         return len(older_perf_data_rows) or self._chunk_size
 
 
+class StalledDataRemoval(RemovalStrategy):
+    """
+    Removes `performance_signature` rows that haven't
```

Comment:
This description needs to be updated since only `performance_datum` rows are removed.